stories/notifications.stories.js

```diff
@@ -217,6 +224,18 @@ storiesOf( 'Global/Notifications', module )
 			type="win-stats"
 		/>
 	) )
+	.add( 'User Input Settings', () => {
+		const setupRegistry = ( registry ) => {
+			registry.dispatch( CORE_USER ).receiveUserInputState( 'missing' );
+			provideSiteInfo( registry );
+		};
+
+		return (
+			<WithTestRegistry callback={ setupRegistry }>
+				<UserInputSettings onCTAClick={ ( e ) => e.preventDefault() } />
```

Comment:
Just because single-letter-variables make me sad: 😉 

```suggestion
				<UserInputSettings onCTAClick={ ( event ) => event.preventDefault() } />
```